smart-contracts/contracts/mixins/MixinGrantKeys.sol

```diff
@@ -20,10 +20,17 @@ contract MixinGrantKeys is
    */
   function grantKeys(
     address[] calldata _recipients,
-    uint[] calldata _expirationTimestamps
+    uint[] calldata _expirationTimestamps,
+    address _keyManager
   ) external
     onlyKeyGranter
   {
+    address keyManager;
+    if (_keyManager != address(0)) {
+      keyManager = _keyManager;
+    } else {
+      keyManager = msg.sender;
+    }
```

Comment:
I want to rework this to support setting the keyManager to 0 intentionally. Same in shareKey.

Reply:
not sure yet how to do it without passing yet another arg though...

Reply:
need to change the else to
```
keyManager = adress(0);
```